event.go

```diff
@@ -46,6 +50,9 @@ type Event struct {
 	Message   string          `json:"message"`
 	// Metadata are structured key-value pairs which describe the event.
 	Metadata map[string]string `json:"meta,omitempty"`
+	// RawMetadata is metadata before it has been transformed to string form.
+	// Metadata and RawMetadata will always contain the same keys.
```

Comment:
It's valid for a caller to construct/mutate their own `Event`. Eg:

```go
ev := slog.Eventf(slog.InfoSeverity, ctx, "foo")
ev.Metadata = map[string]string{
    "bar": "baz"})
// Metadata and RawMetadata have disjoint keys
slog.Log(ev)
```

So I'm not sure there is any guarantee these will always contain the same keys from the perspective of a caller. Maybe change to "should"?

Reply:
Good point. I'll update the comment.